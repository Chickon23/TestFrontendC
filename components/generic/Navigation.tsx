import Link from "next/link";
import React from "react";
import { v4 as uuid } from "uuid";

import { StyledNavigation } from "./navigationStyles";

const Navigation = () => {
  const linkArray = [
    {
      path: "/",
      text: "Home",
    },
    {
      path: "/suche/",
      text: "Suche",
    },
  ];

  return (
    <StyledNavigation className="d-flex justify-content-around align-items-center">
      {linkArray.map(({ path, text }) => (
        <Link key={uuid()} href={path}>
          {text}
        </Link>
      ))}
    </StyledNavigation>
  );
};

export default Navigation;
