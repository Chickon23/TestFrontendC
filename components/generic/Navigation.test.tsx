import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Navigation from "./Navigation";

it("renders homepage unchanged", () => {
  const { container } = render(<Navigation />);
  expect(container).toMatchSnapshot();
});
