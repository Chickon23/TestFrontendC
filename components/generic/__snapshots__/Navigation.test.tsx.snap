// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <nav
    class="sc-bdvvtL igpZwm d-flex justify-content-around align-items-center"
  >
    <a
      href="/"
    >
      Home
    </a>
    <a
      href="/home"
    >
      Stellenanzeigen Home
    </a>
    <a
      href="/suche"
    >
      Suche
    </a>
  </nav>
</div>
`;
